msteams-platform/concepts/deploy-and-publish/apps-upload.md

```diff
@@ -82,18 +82,25 @@ Teams provides several ways to open apps. For more information, see [access your
 
 You don't have to sideload your app again if you make code changes (these are reflected in Teams in real-time). However, you must reinstall if you change any app configurations.
 
-If an update is available to your app, then the **Update available** option is enabled. To update, follow the steps:
+When you update an app in one thread, the new version of the app is automatically updated in all other installed contexts. Following are the steps to update an app:
```

Comment:
When we tell the user that "Following are the steps to update an app", we should also mention that they need not follow all the steps as some apps will auto-install. So the steps to update an app are conditional. What are the conditions? Our dev and admin docs cover it.